runtime/cere/constants/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+// This file is part of Substrate.
```

Comment:
This file (`runtime/cere/constants/src/lib.rs`) should be completely removed. We keep constants in `runtime/common/src/constants.rs` file now. 
The `runtime/cere/constants/src/lib.rs` is out of use and is not imported by any crate currently.